FreeTAKServer/controllers/configuration/MainConfig.py

```diff
@@ -12,12 +13,11 @@ class MainConfig:
 
     # the version information of the server (recommended to leave as default)
     version = 'FreeTAKServer-1.9.9.6 Public'
-    #
-    yaml_path = str(os.environ.get('FTS_CONFIG_PATH', '/opt/FTSConfig.yaml'))
 
+    yaml_path = str(os.environ.get('FTS_CONFIG_PATH', '/opt/FTSConfig.yaml'))
     python_version = 'python3.8'
-
     userpath = '/usr/local/lib/'
+    first_start = True
```

Comment:
first_start must be at the bottom for the setup wizard to change, I assume this is also the case in the ZT